libraries/AP_Beacon/AP_Beacon_Marvelmind.cpp

```diff
@@ -86,20 +83,28 @@ uint8_t AP_Beacon_Marvelmind::mark_position_ready()
     return ind_cur;
 }
 
+uint32_t parse_uint32_le(uint8_t* buffer)
```

Comment:
Ahh was looking for it. I even grepped for bit shifts. 
There is more code relying this operations being done manually. Will provide a PR when finished with this one.